### PR TITLE
Discord embed color settings for join, leave and switch.

### PR DESCRIPTION
**Pull request by loudambiance**
Tested on my server and works as expected:
image

- Version bump to 2.3.0
- Update playerNotification() to take entire Messageutil.MessageReturns message allowing to determine type also allows for easier support of future message types
- Load colors from config or set to default colors in DiscordUtil
- New function to validate that hex code from config.yml is valid
- Removed unused imports (my IDE did this automatically)
- Add IntelliJ IDEA project folder to gitignore
- Update README.md to include info on new config file values

### CODE REVIEW

**Review by OGBlackDiamond — APPROVED** [review]
code looks great, thanks for this.

**Comment by loudambiance** [comment]
Might want to look at it again. I added another commit right before you approved and merged. It added support for {cur} in join and {prev} on leave

**Pull request merged**
